requirements.txt

```
@@ -5,8 +5,7 @@ docutils==0.14
 ebaysdk==2.1.5
 feedparser==5.2.1
 gevent==1.1.2 ; sys_platform != 'win32' and python_version < '3.7'
-gevent==1.3.7 ; sys_platform != 'win32' and python_version >= '3.7'
-gevent==1.4.0 ; sys_platform == 'win32'
+gevent==1.5.0 ; python_version >= '3.7'
```

@mart-e thanks for taking care of this forgotten fw-port.
But I think that there is a problem for our dear Windows users here, if their python version is < 3.7 there is not gevent installed at all. A wheel package exists for both 1.3.7 and 1.5.0 for win32 and x64 platforms.
By the way, Odoo 13.0 follows Debian buster which provides [gevent-1.3.7](https://packages.debian.org/buster/python3-gevent), so we could take advantage of this PR to update it.
```suggestion
gevent==1.3.7 ; python_version < '3.7'
gevent==1.5.0 ; python_version >= '3.7'
```